src/App.css

```diff
@@ -39,52 +68,199 @@
+}
+
+.title {
+  color: #121212;
+  font-size: larger;
+  font-weight: bold;
+}
+
+.author {
+  color: #4386bf;
+}
+
+.buttonsList {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  gap: 1rem;
 }
 
 .buttonsList li {
   list-style: none;
 }
 
-.bookChart {
+.bookBtn {
+  outline: none;
+  border: none;
+  background-color: #fff;
+  color: #4386bf;
+  cursor: pointer;
+  padding-left: 0%;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.progress {
   display: flex;
-  gap: 0.5rem;
-  width: 10rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  min-width: 35%;
 }
 
-.bookChapter {
+.chart {
   display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-  width: 20rem;
+  flex-direction: row;
+  align-items: center;
 }
 
-.addBook {
-  margin-top: 1rem;
+.ProgressBar {
+  margin-right: 10%;
+}
+
+.circle-wrap {
+  width: 75px;
+  height: 75px;
+  background: #e8e8e8;
+  border-radius: 50%;
+  border: 1px solid #cdcbd0;
+}
+
+.mask .fill {
+  clip: rect(0, 37.5px, 75px, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 75px;
+  height: 75px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 75px, 75px, 37.5px);
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(135deg);
+  }
+}
+
+.circle-wrap .inside-circle {
+  width: 61px;
+  height: 61px;
+  border-radius: 50%;
+  background: #fff;
+  line-height: 60px;
+  text-align: center;
+  margin-top: 7px;
+  margin-left: 7px;
+  color: #1e51dc;
+  position: absolute;
+  z-index: 100;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+.percentages {
   display: flex;
   flex-direction: column;
-  gap: 1rem;
-  padding: 0 2rem;
+  margin-left: 5%;
 }
 
-.addBookForm {
-  display: flex;
-  gap: 1rem;
-  margin-bottom: 4rem;
+.percentages h2 {
+  margin-right: 10%;
+  margin-bottom: 0;
+  font-size: 2rem;
+}
+
+.status {
+  opacity: 0.5;
+  margin-top: 0;
+}
+
+.bookChapter {
+  border-left: solid 1px #cdcbd0;
+  padding: 2% 0 2% 8%;
+}
+
+.currentChapter {
+  opacity: 0.5;
+  margin-top: 0;
+}
+
+.updateProgress {
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border: none;
+  border-radius: 3px;
+  background-color: #0290ff;
+  width: 11.5rem;
+  height: 2.063rem;
+  color: #fff;
+  cursor: pointer;
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  margin-top: 1rem;
+}
+
+.addBook {
+  padding: 5%;
+  font-family: 'Montserrat', sans-serif;
+}
+
+form input {
+  width: 23%;
+  padding: 1% 2%;
+  margin-right: 30px;
+}
+
+form select {
+  width: 22%;
+  padding: 1% 2%;
+  margin-right: 30px;
+}
+
+form button {
+  width: 15%;
+  padding: 1% 2%;
+  background-color: #0290ff;
+  border: none;
+  border-radius: 3px;
+  cursor: pointer;
+  color: white;
```

Comment:
- [ ] Please 🙏🏽 align the content of the add form and add the styles assigned to be similar to the Zeplin design.

| Zepplin design | Your design |
|---|---|
| ![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/91c7e9a2-47c6-4ae1-821b-60b09a4e0891) | ![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/9b695aae-67c7-41bd-95d6-22487899b993) |